bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -20,21 +21,24 @@ class IntegTestSuite:
     test_support_matrix.yml
     """
 
-    def __init__(self, component, test_config, bundle_manifest, work_dir):
+    def __init__(self, component, test_config, bundle_manifest, work_dir, test_recorder):
         self.component = component
         self.bundle_manifest = bundle_manifest
         self.work_dir = work_dir
         self.test_config = test_config
+        self.test_recorder = test_recorder
         self.script_finder = ScriptFinder()
         self.repo = GitRepository(
             self.component.repository,
             self.component.commit_id,
             os.path.join(self.work_dir, self.component.name),
         )
+        self.individual_config = None
 
     def execute(self):
         self._fetch_plugin_specific_dependencies()
         for config in self.test_config.integ_test["test-configs"]:
+            self.individual_config = config
```

Comment:
This should be passed down as a parameter, it keeps changing inside an iteration.

Reply:
done